app/scripts/extensions/nav/dropdownMobile.js

```diff
@@ -31,16 +31,37 @@ angular.module('openshiftConsole')
               '</a>',
             '</li>'
           ].join('')
-        }, {
-          type: 'dom',
-          node: _.template([
-            '<li>',
-              '<a href="logout">',
-                '<span class="pficon pficon-user fa-fw" aria-hidden="true"></span>',
-                'Log out <span class="username"><%= userName %></span>',
-              '</a>',
-            '</li>'
-          ].join(''))({userName: (user ? (user.fullName || user.metadata.name) : "") })
         }];
+
+        var launcherApps = Constants.APP_LAUNCHER_NAVIGATION;
+        _.each(launcherApps, function(app) {
+          var liObj = {
+            type: 'dom',
+            node: [
+              '<li>',
+                '<a href="' + _.escape(app.href) + '">',
+                  '<span class="' + app.iconClass + '" aria-hidden="true"></span> ' + app.title,
```

Comment:
Missed on previous review, but we should also call `_.escape` on `iconClass` and `title`